### ISSUE TITLE
No channel for WebGL in documentation

### ISSUE THREAD

**Issue by DylanDevelops** [itchio/butler#243]
I created a game made for being played in the browser. This is my first time pushing an update using butler but looking in the documentation there is no `channel` for webGL. This is what it looks like:

`channel` is which slot you're uploading it to
     for example: `windows-beta`, `osx-bonus`, `linux-universal`, or `soundtrack`

What would I use for WebGL?

Affected Page: https://itch.io/docs/butler/pushing.html

**Comment by duzda** [comment]
Hello, 

As stated in the docs, you can use any name for channel, but using win/linux/mac etc. just automates a few tasks. So all you have to do, is push files to any channel (for example html is a good channel name for build that is supposed to be played in the browser). Then all you have to do is set which file is the browser version on the itch.io website, once set, you can push your updates via butler and it will stay as web build. 

As of right now, you can't push the files straight as the web version, as butler does automatically for you with other builds.

Link to the docs about html games:

https://itch.io/docs/butler/pushing.html#html--playable-in-browser-games

Here's what you have to do manually on the itch.io website. I have chosen the name of the channel as html, this doesn't make anything automatically, this is just for me so I know this channel is used for web build.
![image](https://user-images.githubusercontent.com/25201406/197964257-a2f55351-0a21-45f1-8d85-83a5a4be4114.png)

This is what my game's channels look like in butler. I can push a new update to html channel and it will be automatically playable on the website in a few minutes, since I set it on the website.
![image](https://user-images.githubusercontent.com/25201406/197964312-49d28b2d-1c72-4d1c-ae39-45b1b06b939f.png)

By no means am I saying this wouldn't be a wonderful feature request, just wanted to note that docs do specify how to upload html games. And hopefully help you or anyone else who stumbles upon this.

**Comment by DylanDevelops** [comment]
Yeah, thank you for the info. I figured it out a while back but that does make more sense.
